Makefile

```diff
@@ -31,6 +31,14 @@ image: build
 push: image
 	docker push quay.io/rhacs-eng/qa:cve-uploader-$(TAG) | cat
 
+.PHONY: tag-latest
+tag-latest: image
+	docker tag quay.io/rhacs-eng/qa:cve-uploader-$(TAG) quay.io/rhacs-eng/qa:cve-uploader-latest
```

Comment:
I really think we shouldn't abuse `quay.io/rhacs-eng/qa` and `quay.io/rhacs-eng/cve-uploader` will be a better thing, even at transitional time. That's partially on me. I promised a HOWTO article that I'll create now.
Would you be ok if I assist creating a separate repo for cve uploader images?

Reply:
I didn't want to bother making yet another repo to maintain just for temporary purposes, but if people are ok with it, then it works for me. I'll follow the steps you outlined and bother you with a repo request :)